libtiledbvcf/src/read/reader.h

```diff
@@ -495,16 +499,24 @@ class Reader {
 
   /**
    * Prepares sample batches for export
-   * @return
+   * @param all_samples Boolean to indicated if all samples are being queried by
```

Comment:
I would have the word "output" somewhere in this description , same for the L515.